tests/integration_test.rs

```diff
@@ -122,3 +189,23 @@ fn assert_ok_response(response: Result<ureq::Response, ureq::Error>) -> ureq::Re
 
     response
 }
+
+fn assert_response_status_code(response: Result<ureq::Response, ureq::Error>, expected_code: u16) {
```

Comment:
I was initially confused why this assertion would panic on a ok response. What about changing its name to same thing like `assert_response_error`?